.github/workflows/auto-build-rpm.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: |
           export VERSION=${{ steps.branch_env.outputs.version }}
           sudo gem install --no-document fpm
-          git clone https://github.com/api7/apisix-build-tools.git
+          git clone https://github.com/FangSen9000/apisix-build-tools.git
```

Comment:
Note that we should let the API7 community merge my changes to the build script before merging this pr.
https://github.com/api7/apisix-build-tools/pull/232
(if I keep it as it is, the original warehouse that has not been changed will only report an error. In order to prove that my changes are effective, I link to my modified build script warehouse.)